MostlyArmless/ChatApp
# Fixing intro message handshake

Changed postincrement to preincrement so now the first client to connect is assigned clientId of 1 instead of 0.

To test:
1. start server
2. open SWC chrome extension
3. connect to server with SWC and send a valid clientIntroMessage (expect response that assigns clientId of 1)
4. start frontend, open console in browser (expect to see a message from the server that assigned clientId of 2)
